storyboarding/scripting/compound_commands.rst

```diff
@@ -12,6 +12,405 @@ Loop
+  :scale: 20%
+  :alt: One of Aoba's pigtails
+  :align: left
+
+.. image:: img/compound_commands/drum.png
+  :scale: 30%
+  :alt: The taiko drum
+  :align: left
+
+First of all we are moving our static sprites into place:
+
+.. code-block:: c
+  :linenos:
+  :caption: Our static sprites
+  :name: Static sprites
+
+  Sprite,Background,Centre,"sb\aobaHair.png",192.625,140
+   S,0,0,,0.2
+   F,0,0,,1
+  Sprite,Background,CentreLeft,"sb\drum.png",186,163
+   S,0,0,,0.3
+   F,0,0,,1
+
+When nothing is happening the sprite of the idling Aoba should be shown. This will be the case at the start of the beatmap.
+As soon as any hitsound is played she should get replaced by one of the other two sprites.
+
+.. code-block:: c
+  :linenos:
+  :caption: Aoba preparing to hit the Taiko
+  :name: IdleCode
+
+  Sprite,Background,Centre,"sb\aobaTaikoIdle.png",192.625,140
+   F,0,0,,1
+   S,0,0,,0.2
+   F,0,105330,,0
+   T,HitSound,0,105330
+    F,0,0,,0
+    F,0,110,,1
+
+Fairly simple, isn'it? Now let's do the same for the other two sprites except that they are invisible at the start and fade in on the according hitsound.
+
+.. code-block:: c
+  :linenos:
+  :caption: Aoba hitting the drum with passion!
+  :name: MovingCode
+
+  Sprite,Background,Centre,"sb\aobaTaikoKat.png",192.625,140
+   F,0,0,,0
+   S,0,0,,0.2
+   T,HitSoundWhistle,0,105330
+    F,0,0,,1
+    F,0,110,,0
+   T,HitSoundClap,0,105330
+    F,0,0,,1
+    F,0,110,,0
+  Sprite,Background,Centre,"sb\aobaTaikoDon.png",192.625,140
+   F,0,0,,0
+   S,0,0,,0.2
+   T,HitSound4,0,105330
+    F,0,0,,1
+    F,0,110,,0
+
+And that's it! Aoba will now play the drum in sync with the map!
+You can see the application of this effect in the following map: https://osu.ppy.sh/s/597411
+
+.. warning:: Yeah, the map, not with the player. Hitsound-triggers are only activated by Hitsounds from objects. Otherwise this would be amazing for Taiko-mode effects but you can still do some interesting things with it!
+  The other sad thing is that spinners and drumrolls (except for the head) in Taiko don't trigger hitsounds but you could perfectly let Aoba drum to an osu!standard difficulty.
+  This solution is far from universal as you will see in the Pitfalls-section but it works perfectly fine for Kantans and most Futsuu difficulties.
+
+Change of Gamestate
+~~~~~~~~~~~~~~~~~~~
+.. it is safe to say that Passing/Failing triggers literally don't work in CtB and Mania
+  attention:: If you are planning to use Passing and Failing triggers in a map for either Catch the Beat or mania, go no further because these are literally not implemented.
+  There is full support for standard and partial support for Taiko (only Don/Kat, no drumrolls/spinners). Keep this in mind before making big plans with this type of trigger.
+
+As you might know already from reading the chapter about layers and objects, osu! supports 4 layers of which 2 are `Pass` and `Fail` that are displayed in accordance to the current game state.
```

Comment:
Try condensing this given the tips I've told you about prior. :)

Reply:
Back in english class >.<"
Should be better now, please be gentle with grading, Kirby-sensei.